website/source/api/acl/policies.html.md

```diff
@@ -69,8 +61,9 @@ The table below shows this endpoint's support for
 
 ### Sample Request
 
-```sh
-$ curl -X PUT \
+```text
+$ curl \
+    --request PUT \
```

Comment:
Did you mean to change this to `text`? And I would prefer to keep `-X` because it is more common. I know `--request` is the long version.

Reply:
There are multiple changes like that further down.

Reply:
Looks like a bad rebase.

Give me some time and I'll go over it again.